### PR TITLE
Update README.md for solving issue #1290

### LINKED ISSUE #1290: ➕ [RESOURCE] - For finding researchpaper on computer engineering
### Idea Contribution

- [X] I have read all the resource request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Where do you want to add the resource? Specify the category and section.

Computer engineering- Resource for researchpaper

### Provide the URLs of the resources you are adding

ACMDigital Library

### Any alternative approaches/features

_No response_

### Additional Context

_No response_

### Code of Conduct

- [X] I follow the [CONTRIBUTING GUIDELINE](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) of this project.

### PR DESCRIPTION
**Pull request by Aditi22Bansal**
# Pull Request

## Description
Please include a summary of the changes and the related issue.  

added a resource 

Closes #1290 

## Type of change
- [x] Adding a new Resource
- [ ] Adding a new Category
- [ ] Documentation enhancement or fixes
- [ ] Website enhancement or fixes

## Checklist:
- [x] I read carefully [CONTRIBUTING.md](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Screenshots (if appropriate):

Please add screenshots to help explain your changes.
image

### CODE REVIEW

**Comment by Aditi22Bansal** [comment]
@jfmartinz 
Kindly review and merge the PR

File: Computer Engineering/README.md [hunk 1]
```diff
@@ -260,6 +260,16 @@ Welcome to the Computer Engineering resource repository! This collection offers
...
+<table width="100%">
+  <tr>
+    <th>Resource Name</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><a href="https://dl.acm.org/">ACM Digital Library</a></td>
+    <td>Hosted by the Association for Computing Machinery, this digital library offers a comprehensive collection of papers, articles, and conference proceedings in the field of computer science and engineering.</td>
+  </tr>
+</table>
```
Comment (jfmartinz) [inline Computer Engineering/README.md:272]:
```suggestion
  <tr>
    <td><a href="https://dl.acm.org/">ACM Digital Library</a></td>
    <td>Hosted by the Association for Computing Machinery, this digital library offers a comprehensive collection of papers, articles, and conference proceedings in the field of computer science and engineering.</td>
```

Moved this table row into this, dont create another table for it. 

https://github.com/jfmartinz/ResourceHub/blob/4b3a99553c3494f92016daf642f4d68d1c336a40/Computer%20Engineering/README.md#resources-for-finding-research-papers

**Review by jfmartinz — CHANGES_REQUESTED** [review]

File: Computer Engineering/README.md (hunk 1 above)
Comment (Aditi22Bansal) [reply Computer Engineering/README.md:272]: ok

**Comment by Aditi22Bansal** [comment]
@jfmartinz 
Done
Kindly review and merge

**Review by jfmartinz — APPROVED** [review]

**Pull request merged**